src/coreclr/ToolBox/superpmi/superpmi-shared/methodcontext.h

```diff
@@ -43,6 +43,15 @@ const char* toString(CorInfoType cit);
 
 #define METHOD_IDENTITY_INFO_SIZE 0x10000 // We assume that the METHOD_IDENTITY_INFO_SIZE will not exceed 64KB
 
+// Special "jit flags" for noting some method context features
+
+enum EXTRA_JIT_FLAGS
+{
+    HAS_PGO = 63,
```

Comment:
I wonder if you should put a note in corjitflags.h and jitee.h that you're doing this. Or, put a reference here (like a static compile-time assert) using `JIT_FLAG_UNUSED34`, `JIT_FLAG_UNUSED35`, `JIT_FLAG_UNUSED36`, so if anyone changes those, we'll get a build error here.

Reply:
Added.